.travis.yml

```diff
@@ -83,6 +83,11 @@ script:
   - |
     if [[ -v PRODUCT_TESTS_SPECIFIC_ENVIRONMENT ]]; then
       presto-product-tests/bin/run_on_docker.sh \
+        singlenode-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation
```

Comment:
@arhimondr I wasn't really sure if I need to include the `authorization` group here. If it includes `authorization`, there is also have some other errors after adding the patch.

Reply:
No. You don't need authorization there. Authorization is disabled on a default image.

Reply:
I tried to remove `authorization`. It seems the CI won't fail then. Trying to modify the impersonation test. Not really sure I'm doing it correctly.